courses/machine_learning/deepdive2/introduction_to_tensorflow/labs/feat.cols_tf.data.ipynb

```diff
@@ -19,7 +19,9 @@
     "\n",
```

Comment:
<p>There is no objective indicated. Please return the code as is.  Thanks!</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/GoogleCloudPlatform/training-data-analyst/pull/1089/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='41'/>

Reply:
Thanks @gstripling ,
Updated the cell with as before.